### ISSUE TITLE
Request Add Install.RDF target for Basilisk Browser, Thank you

### ISSUE THREAD

**Issue by Red-Eagle-LXIX** [JustOff/responsive-bookmarks-toolbar#2]
Request if willing, could you add Install.rdf target application for Basilisk?
I successfully installed and have this addon working with Basilisk by copying the extension from my Palemoon instal and hacking the install.rdf.
Functionality is perfect in Basilisk with Classic Theme Restorer (untested without simply as another addon I use requires CTR).
I'm trying to keep both my Palemoon and Basilisk installs as close as possible and this is one excellent addon.

**Comment by bezik46** [comment]
```
   <!-- Firefox -->
    <em:targetApplication>
      <Description>
        <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
        <em:minVersion>52.0</em:minVersion>
        <em:maxVersion>56.*</em:maxVersion>
      </Description>
    </em:targetApplication>
```

**Comment by JustOff** [comment]
Resolved by [2.0.4](https://github.com/JustOff/responsive-bookmarks-toolbar/releases/tag/2.0.4).
